.gitignore

```diff
@@ -1,5 +1,9 @@
 .DS_Store
 
+#Lock files
+package-lock.json
+yarn.lock
+
```

Comment:
Alguma razão em especial para ignorar os arquivos de lock? Até onde eu sei, estes arquivos impedem que mais de um desenvolvedor utilize dependencias de versoes diferentes.  Veja esta thread para mais info: https://stackoverflow.com/questions/44206782/do-i-commit-the-package-lock-json-file-created-by-npm-5 :)

Reply:
Gerencia de dependencia é fonte de tanta treta que tem toda uma área de pesquisa em eng. de software encima disso, vc acredita? :) tamo junto!